packages/pages/src/components/image/image.tsx

```diff
@@ -70,47 +73,89 @@ export const Image = ({
     .map((w) => `${getImageUrl(imgUUID, w, (imgHeight / imgWidth) * w)} ${w}w`)
     .join(", ");
 
-  const onError = () => {
-    setImgStatus(imgLoadingStatus.ERROR);
-    console.warn(`Invalid image src: ${src}.`);
-  };
-
   return (
     <>
-      {placeholder != null &&
-        imgStatus != imgLoadingStatus.LOADED &&
-        placeholder}
-      {imgStatus != imgLoadingStatus.ERROR && (
+      {!imgLoaded && placeholder != null && placeholder}
+      {src && (
```

Comment:
I don't think the src check is necessary. It's always set by the time we get here.